fabric_cf/orchestrator/core/orchestrator_slice_wrapper.py

```diff
@@ -127,11 +127,27 @@ def create(self, *, bqm_graph: Neo4jCBMGraph, slice_graph: Neo4jASM,
             slivers = []
             for nn_id in slice_graph.get_all_network_nodes():
                 sliver = slice_graph.build_deep_node_sliver(node_id=nn_id)
+
+                # Compute Requested Capacities from Capacity Hints
+                requested_capacities = sliver.get_capacities()
+                requested_capacity_hints = sliver.get_capacity_hints()
+                catalog = InstanceCatalog()
+                if requested_capacities is None and requested_capacity_hints is not None:
```

Comment:
Seems checking for requested_capacity_hints is not None is redundant? Basically you are prioritizing capacity over capacity_hints, which I agree with, correct?